crates/solver-service/src/apis/quote/generation.rs

```diff
@@ -130,6 +137,7 @@ impl QuoteGenerator {
 			quote_id,
 			provider: "oif-solver".to_string(),
 			cost: None,
+			lock_type,
```

Comment:
Mind also pushing this change to oif-specs?

cc @pepebndc

Reply:
I'm not sure if it needs to be different from what we store locally just yet.. how much do we differ from the current response (solver local vs solver to aggregator/user)